contracts/wrappers/SetStepFunctionWrapper.sol

```diff
@@ -25,22 +26,32 @@ contract SetStepFunctionWrapper is Withdrawable {
     }
 
     function setImbalanceStepFunction(ERC20 token,
```

Comment:
shift the `ERC20 token` to the next line?

Reply:
done